packages/pixeloven-exceptions/src/Node/index.ts

```diff
@@ -0,0 +1 @@
+export { default as NodeProcessException } from "./ProcessException";
```

Comment:
Did you mean to update this exported name to match the new class name in the file above??

Reply:
NVM, I see what you're doing with the dir structure now

Reply:
Eventually I want to do Node.ProcessException but that would require me to make other changes that I wasn't willing to make yet

Reply:
I had a feeling that's where you're headed looking at all the dirs as a collection.